src/test/rpc/AccountObjects_test.cpp

```diff
@@ -579,19 +579,35 @@ class AccountObjects_test : public beast::unit_test::suite
                 (resp[jss::result][jss::account_objects].size() == size);
         };
 
+        // Make a lambda that checks if the response has error for invalid type
+        auto acct_objs_type_is_invalid = [](Json::Value const& resp) {
```

Comment:
I respect and honor that you're following file-local naming conventions by naming this lambda in snake case.  You're following the model of the `acct_objs` and `acct_objs_is_size` lambdas just above.

Unfortunately the above lambda, `acct_objs_is_size`, does not follow the general guidance in this code base that functions are named in `camelCase` with an initial lower case letter.  So I encourage you to rename your new lambda to `acctObjsTypeIsInvalid`.  If you're feeling really helpful, you could also consider renaming...

- `acct_objs`,
- `acct_objs_is_size`, and
- `scenv_acct_objs`

to use `camelCase` as well.  That would help to stem the spread of the rot.  But I also understand that you may not want to extend the scope of the pull request.